src/test/java/com/karankumar/bookproject/backend/statistics/PageStatisticsTest.java

```diff
@@ -74,22 +76,22 @@ void onlyReadBooksCountTowardsMostPagesStatistics() {
     @Test
     void testAveragePageLengthDivideByZero() {
         resetPageStatistics();
-        assertThat(pageStatistics.calculateAveragePageLength()).isNull();
+        assertTrue(pageStatistics.calculateAveragePageLength().isEmpty());
```

Comment:
In line with our style guide, could you please replace this with `assertThat`? We're using AssertJ assertions